apps/web/src/components/NavContext/NavContextContainer.tsx

```diff
@@ -1,8 +1,10 @@
-import { createContext, ReactNode, useMemo, useState } from "react";
+import { createContext, ReactNode, useMemo } from "react";
 
 import { RoleName } from "@gc-digital-talent/auth";
 import { assertUnreachable } from "@gc-digital-talent/helpers";
-import { useLogger } from "@gc-digital-talent/logger";
+import { useLocalStorage } from "@gc-digital-talent/storage";
+
+import { NAV_ROLE } from "../../../../../packages/auth/src/const";
```

Comment:
```suggestion
import { NAV_ROLE } from "@gc-digital-talent/auth/const";
```

Reply:
Fixed in [c0a75d8](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11501/commits/c0a75d801f67b24a7a4ba7e113ff1ee664c2e0bd)